canonical-web-and-design/vanilla-framework
# Update notifications component with new style and features

## Done

This is a feature branch that gathers the work on new notification style

## QA

- Open [demo](https://vanilla-framework-3856.demos.haus/docs/patterns/notification)
- Make sure all work as expected
- This is a feature branch, individual PRs have been reviewed and QAed already, so just a final verification

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical-web-and-design/vanilla-framework/wiki/Release-process#pre-release-tasks)
